assets/js/script.js

```diff
@@ -0,0 +1,32 @@
+const button = document.querySelector('.menu');
+const menu = document.querySelector('.desktop');
+const body = document.querySelector('body');
+const close = document.querySelector('.x');
+const closePortfolio = document.querySelector('.link');
+const closeAbout = document.querySelector('.link2');
+const closeContact = document.querySelector('.link3');
+
+button.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  body.classList.toggle('active');
+});
+
+close.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  body.classList.toggle('active');
+});
+
+closePortfolio.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  body.classList.toggle('active');
+});
+
+closeAbout.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  body.classList.toggle('active');
+});
+
+closeContact.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  body.classList.toggle('active');
+});
```

Comment:
- [ **OPTIONAL** ] I also think that it would be better if you remove or cancel the scrolling behavior on the body content when the **mobile-menu** appears so you can not interact with the body content when you are in the **mobile menu mode**. i.e `body.active{ overflow: hidden }`. But good job :100:

Reply:
Great idea, I didn't know that was possible